pipeline.py

```diff
@@ -0,0 +1,108 @@
+import argparse
+import json
+import os
+import sys
+
+from bgp import ia, DEFAULT_SEQUENCER
+
+# TODO
+# Hit OL for ISBN info and save in folder
+# Look into sqlite3
+
+parser = argparse.ArgumentParser(prog='[pipeline]',
+                                 description='Automate Open Book Genome Project sequencer')
+
+parser.add_argument('Path',
+                    metavar='source-path',
+                    type=str,
+                    help='path to the jsonl file with ia identifiers')
+
+args = parser.parse_args()
+
+input_path = args.Path
+
+if not os.path.isfile(input_path):
+    print('The path specified does not exist')
+    sys.exit()
+
+RESULTS_PATH = 'results/bgp_results/'
+books = []
+
+
+def touch(identifier, record):
+    file_name = '{}{}/{}_{}'.format(RESULTS_PATH, identifier, record, identifier)
+    if os.path.exists(file_name):
+        raise Exception('DatabaseRecordConflict')
+    else:
+        open(file_name, 'a').close()
+
+
+def db_isbn_extracted(identifier, isbn):
+    touch(identifier, 'ISBN_{}'.format(isbn))
+
+
+def db_isbn_none(identifier):
+    touch(identifier, 'UPDATE_NONE')
+
+
+def db_update_failed(identifier):
+    touch(identifier, 'UPDATE_FAILED')
+
+
+def db_update_succeed(identifier):
+    touch(identifier, 'UPDATE_SUCCEED')
+
+
+def db_update_conflict(identifier):
+    touch(identifier, 'UPDATE_CONFLICT')
```

Comment:
Some of these methods may need to be slightly more clever.

e.g. What happens if there's a `UPDATE_SUCCEED` on disk and we try to run the pipeline again? Or what if `UPDATE_FAILED` is on disk and we re-run?

Reply:
If we run the pipeline again those items should be skipped because there are already directories for them. If we want them to be resequenced then we would delete the item directories.